Design_Systems/Lightning/DetailView.xml

```diff
@@ -0,0 +1,338 @@
+												<skuid__tabPanel name="Related">
+													<components/>
+												</skuid__tabPanel>
+												<skuid__tabPanel name="Details">
+													<components>
+														<skuid__wrapper uniqueid="sk-2fOY-45855">
+															<components>
+																<skuid__form showErrorsInline="true" model="Account" uniqueid="sk-2aFi-33158" mode="read">
+																	<columns columnGutter="24px">
+																		<column behavior="flex" verticalAlign="top">
+																			<sections>
+																				<section title="New Section" showHeading="false">
+																					<fields>
+																						<skuid__field id="OwnerId" uniqueId="sk-2aMu-78248" tokenizeSearch="false"/>
+																						<skuid__field id="Name" uniqueId="sk-2aMu-78261"/>
+																						<skuid__field id="ParentId" uniqueId="sk-2aMu-78274"/>
+																						<skuid__field id="Site" uniqueId="sk-2aMu-78300"/>
+																						<skuid__field id="Type" uniqueId="sk-2aMu-78313"/>
+																						<skuid__field id="Industry" uniqueId="sk-2aMu-78326"/>
+																						<skuid__field id="AnnualRevenue" uniqueId="sk-2aMu-78339"/>
+																						<skuid__field type="COMBO" label="Billing Address" uniqueId="sk-2ei0-27606" template="{{BillingStreet}}&lt;br&gt;&#10;{{BillingCity}}, {{BillingState}} {{BillingPostalCode}}" allowHtml="true"/>
+																						<skuid__field id="CreatedById" uniqueId="sk-2aMu-78365"/>
+																						<skuid__field id="CreatedDate" uniqueId="sk-2aMu-78378"/>
+																					</fields>
+																				</section>
+																			</sections>
+																		</column>
+																		<column behavior="flex" ratio="1" width="200px" verticalAlign="top" minWidth="100px">
+																			<sections>
+																				<section title="New Section" showHeading="false">
+																					<fields>
+																						<skuid__field id="Rating" uniqueId="sk-2aN5-82464"/>
+																						<skuid__field id="Phone" uniqueId="sk-2aN5-82477"/>
+																						<skuid__field id="Fax" uniqueId="sk-2aN5-82490"/>
+																						<skuid__field id="Website" uniqueId="sk-2aN5-82503"/>
+																						<skuid__field id="TickerSymbol" uniqueId="sk-2aN5-82516"/>
+																						<skuid__field id="NumberOfEmployees" uniqueId="sk-2aN5-82542"/>
+																						<skuid__field id="ShippingAddress" uniqueId="sk-2aN5-82568"/>
+																						<skuid__field id="LastModifiedById" uniqueId="sk-2aN5-82581"/>
+																						<skuid__field id="LastModifiedDate" uniqueId="sk-2aN5-82594"/>
+																					</fields>
+																				</section>
+																			</sections>
+																		</column>
+																	</columns>
+																	<styles>
+																		<spacing top="1.5" bottom="1.5" left="1.5" right="1.5"/>
+																	</styles>
+																</skuid__form>
+															</components>
```

Comment:
Can we get a save / cancel button with the form.  Just so the page is slightly functional?

Reply:
If we note that the page isn't functional, do you still want the save/cancel buttons added and wired up?

Reply:
No - we can do without them.  I'll resolve this conversation.